tests/test_datasets/test_dataset.py

```diff
@@ -266,18 +291,22 @@ def ts_without_target_components():
 
 
 @pytest.fixture
-def ts_with_target_components():
+def ts_with_target_components(target_components_df):
     timestamp = pd.date_range("2021-01-01", "2021-01-15")
-    df_1 = pd.DataFrame(
-        {"timestamp": timestamp, "target": 3, "target_component_a": 1, "target_component_b": 2, "segment": 1}
-    )
-    df_2 = pd.DataFrame(
-        {"timestamp": timestamp, "target": 7, "target_component_a": 3, "target_component_b": 4, "segment": 2}
-    )
+    df_1 = pd.DataFrame({"timestamp": timestamp, "target": 3, "segment": 1})
+    df_2 = pd.DataFrame({"timestamp": timestamp, "target": 7, "segment": 2})
     df = pd.concat([df_1, df_2])
-    df = TSDataset.to_dataset(df)
+    df = TSDataset.to_dataset(df=df)
+
     ts = TSDataset(df=df, freq="D")
-    ts._target_components_names = ["target_component_a", "target_component_b"]
+    ts.add_target_components(target_components_df=target_components_df)
+    return ts
+
+
+@pytest.fixture()
+def ts_with_prediction_intervals(ts_without_target_components, prediction_intervals_df):
+    ts = deepcopy(ts_without_target_components)
```

Comment:
I'm not sure that it is necessary to make `deepcopy` in fixtures.

Reply:
If we don't do so, this fixture will modify `ts_without_target_components` inplace. But we want it untouched.

Reply:
I thought that fixtures doesn't interact with each other during creation.